### PR TITLE
fix build issues for Android

### PR DESCRIPTION
**Pull request by sandstranger**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -50,27 +55,39 @@ include(OpenMWMacros)
 
 configure_file ("${OpenMW_SOURCE_DIR}/docs/mainpage.hpp.cmake" "${OpenMW_BINARY_DIR}/docs/mainpage.hpp")
 
-option(MYGUI_STATIC "Link static build of Mygui into the binaries" FALSE)
-option(BOOST_STATIC "Link static build of Boost into the binaries" FALSE)
+if (ANDROID)
+    set (STATIC_BUILD TRUE)
+else()
+    set (STATIC_BUILD FALSE)
+endif()
```
Comment (scrawl) [inline CMakeLists.txt]: This change is not needed.

File: CMakeLists.txt [hunk 2]
```diff
@@ -50,27 +55,39 @@ include(OpenMWMacros)
...
 option(OSG_STATIC "Link static build of OpenSceneGraph into the binaries" FALSE)
 option(QT_STATIC "Link static build of QT into the binaries" FALSE)
 
 option(OPENMW_UNITY_BUILD "Use fewer compilation units to speed up compile time" FALSE)
 
+if (ANDROID)
+    set (NEED_BUILD OFF)
+else()
+    set (NEED_BUILD ON)
+endif()
```
Comment (scrawl) [inline CMakeLists.txt]: This change is not needed.

File: cmake/FindMyGUI.cmake [hunk 3]
```diff
@@ -71,7 +71,7 @@ ELSE (WIN32) #Unix
     FIND_PACKAGE(PkgConfig)
     IF(MYGUI_STATIC)
         # don't use pkgconfig on OS X, find freetype & append it's libs to resulting MYGUI_LIBRARIES
-        IF (NOT APPLE)
+        IF (NOT APPLE AND NOT ANDROID)
```
Comment (scrawl) [inline cmake/FindMyGUI.cmake]: Why do you not want to use pkgconfig for android?

Comment (sandstranger) [inline cmake/FindMyGUI.cmake]: It seems my local pkgconfig file for mygui  broken , I will try to fix issue

**Comment by sandstranger** [comment]
So , issue with mygui was in my local pkg file , I will add another PR , because it contains unnecessary changes

**Pull request closed without merging**